tests/dbt/test_transformation.py

```diff
@@ -410,6 +410,40 @@ def test_seed_columns():
     assert sqlmesh_seed.column_descriptions == expected_column_descriptions
 
 
+def test_seed_column_types():
+    seed = SeedConfig(
+        name="foo",
+        package="package",
+        path=Path("examples/sushi_dbt/seeds/waiter_names.csv"),
+        column_types={
+            "address": "text",
+            "zipcode": "text",
+        },
+        columns={
+            "zipcode": ColumnConfig(name="zipcode", description="Business zipcode"),
+        },
+        quote_columns=True,
+    )
+
+    expected_column_types = {
+        "address": parse_one("text", into=exp.DataType),
```

Comment:
Open question: In terms of using SQLGlot, is the preferred way to construct `DataType` objects vs `exp.DataType.build("text")`?

Reply:
We should use `exp.DataType.build`, that's the prevalent pattern in the codebase.

Reply:
sure I will change these and make sure to use this way from now on